src/push.go

```diff
@@ -86,7 +86,18 @@ func PushManyWithGitImpl(ctx context.Context, flags *PushFlags, repoNames []stri
 }
 
 func PushWithGitImpl(ctx context.Context, flags *PushFlags, repoName string, ghClient *github.Client, gitimpl GitImplementation) error {
-	_, nwo, err := extractSourceDest(repoName)
+	org, nwo, err := extractSourceDest(repoName)
+	if err != nil {
+		return err
+	}
+
+	orgOwnerName, orgRepoName, err := splitNwo(org)
```

Comment:
```suggestion
	sourceOwnerName, sourceRepoName, err := splitNwo(source)
```

Reply:
Sure but if we change `org` to `source`, there are a bunch of changes to add in the code, not related to my PR and we break the global logical naming rules in the project. Are you sure about it ?

Here's an example of existing code in the same file which use `org` to refer about

```go
// check if the owner refers to the authenticated user or an organization.
var createRepoOrgName string
if strings.EqualFold(*currentUser.Login, ownerName) {
    // we'll create the repo under the authenticated user's account.
    createRepoOrgName = ""
} else {
    // ensure the org exists.
    createRepoOrgName = ownerName
    _, err := getOrCreateGitHubOrg(ctx, client, ownerName, *currentUser.Login)
    if err != nil {
	return nil, err
    }
}
```

Reply:
Done !